var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -43,11 +41,7 @@ class Relion(CMakePackage, CudaPackage):
     depends_on('fltk', when='+gui')
     depends_on('libtiff')
 
-    # relion 3 supports cuda 9
-    # relion < 3 does not
     depends_on('cuda', when='+cuda')
-    depends_on('cuda@9:', when='@3: +cuda')
-    depends_on('cuda@8.0:8.99', when='@:2 +cuda')
```

Comment:
You say you remove the old versions because they are unstable, but what if someone only has CUDA 8 installed on their cluster? Will the latest version only work with CUDA 9+?

Reply:
You are correct per usual

It requires cuda 9 or above.

```

    cuda@8


while trying to concretize the partial spec:

    relion%gcc@7.4.0 build_type=Benchmarking cuda_arch=70
        ^cmake@3:3.9.4
        ^fftw+double+float
        ^libtiff
            ^jpeg
            ^xz
            ^zlib
        ^openmpi+cuda fabrics=verbs
            ^hwloc@:1.999+cuda
                ^cuda@8
                ^pkgconfig
            ^numactl
                ^autoconf
                    ^m4@1.4.6:
                    ^perl
                        ^gdbm
                            ^readline
                                ^ncurses
                ^automake
                ^libtool


```

Reply:
Is there any reason to remove the older versions of `relion`?

Reply:
The community does not use them because the performance is much slower and they're filled with bugs.